color_refiment_helper.py

```diff
@@ -92,7 +98,7 @@ def get_vertices_of_graph(partition: List["Vertex"], g: "Graph") -> List["Vertex
 def are_twins(u: "Vertex", v: "Vertex") -> bool:
     N_u = [x for x in u.neighbours if x != v]
     N_v = [x for x in v.neighbours if x != u]
-    return compare(N_u,N_v)
+    return compare(N_u,N_v, lambda vertex: vertex.label)
```

Comment:
werkt `mykey=vertex.label` ook?
Of `mykey= Vertex.label`
Of anders `mykey=Vertex.__str__`
Pure nieuwsgierigheid btw.

Reply:
Nup, dat kan in ieder geval niet als het als param mee wil geven. Dan gaat het heel erg over de zeik :)